examples/quickstart/jupyter-notebooks/druid-visualization.ipynb

```diff
@@ -0,0 +1,620 @@
+    "The following cell takes the JSON results from the Druid API and uses the pandas [`read_json`](https://pandas.pydata.org/docs/reference/api/pandas.read_json.html) method to convert the results to a pandas object. The pandas ojbect lets us display the results in a tabular format. The `orient` parameter sets pandas to accept data as a list of records in the format: `[{column: value, ...}, {column: value, ...}]`.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c3860d64-fba6-43bc-80e2-404f5b3b9baa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "additions_pd = pd.read_json(json.dumps(response.json()), orient='records')\n",
+    "\n",
+    "additions_pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8c8f6e17-aa3e-4871-94d8-e4bed74ed516",
+   "metadata": {},
+   "source": [
+    "You can also load your results into a pandas DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "67e6e09f-97ed-4b7c-b007-d239010f069c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "additions_df = pd.DataFrame.from_records(additions_pd)\n",
+    "\n",
+    "additions_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9488cbee-63bf-414a-aff3-207adf39e115",
+   "metadata": {},
+   "source": [
+    "One benefit of using a DataFrame is that you can access all the pandas `DataFrame` object methods. For example, `DataFrame.max()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4d0a4ada-c9cf-4eee-a55f-b524d2dfad64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "additions_df.max()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1720a88-cf1b-4022-981a-69441fac54fd",
+   "metadata": {},
+   "source": [
+    "Check out the [pandas docs](https://pandas.pydata.org/docs/getting_started/intro_tutorials/index.html) for more ideas about how to use pandas and DataFrames \n",
+    "with your Druid data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07a16047-dba1-4f45-9fcb-6130c37eca61",
+   "metadata": {},
+   "source": [
+    "## Display data with a bar graph\n",
+    "\n",
+    "Tabular format is OK for scanning data, however you can use visualization and plotting tools that work with Jupyter notebooks to visualize data as plots or graphs. This section uses [Bokeh](https://bokeh.org/) to illustrate some basic plots using Druid data.\n",
+    "\n",
+    "In this section, you create a simple bar chart showing the channels with the most additions during the time range."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8420df27-0e80-48d1-a806-43f6bf81dd16",
+   "metadata": {},
+   "source": [
+    "First, call `output_notebook()` to set Bokeh to output plots inline within the notebook. This lets you view your plot inline. You can also use Bokeh's `output_file` function to write your plot to an HTML file. If you want to experiment with `output_file` add it to the list of imports."
```

Comment:
```suggestion
    "First, call `output_notebook()` to set Bokeh to output plots inline within the notebook. This lets you view your plot inline. You can also use Bokeh's `output_file` function to write your plot to an HTML file. If you want to experiment with `output_file`, add it to the list of imports."
```
What about adding something like this to the end of the paragraph:

`````
For example:
    ```
    from bokeh.io import output_notebook, show, output_file
    ```
`````